### DISCUSSION TITLE
provide a demo(dummy) data to see and learn concepts like facets and collections #1157

### DISCUSSION THREAD

**Discussion by loveyandex** [vendurehq/vendure#1159] [General]
please show us power of vendure with sample data to add it in database not in demo storefront
thanks

**Comment by michaelbromley** [comment]
Hi,

I'm not sure what you are asking for here. The demo and the `@vendure/create` package already includes sample data with facets and collections.

**Reply by loveyandex** [reply]
@michaelbromley 
Channels
Job Queue
Promotions
Payment Integrations
Stock Control
Taxes

we have no idea about them sample data and making query can help us

why u use graphql???? 
it is really hard to use and have a many waist time with no profits vs rest
we must learn graphql , appolo modules , nestjs graphql resolving and many attemps reduce our speed to boost applications

**Reply by michaelbromley** [reply]
> we have no idea about them sample data and making query can help us

The sample data already includes stock control, taxes, and a sample payment integration, and makes use of the job queue. But yes perhaps it would be good to include a sample promotion and channel.

> why u use graphql????

I personally find GraphQL very nice and productive to work with. I know there is extra to learn, but that's the trade-off. If you very much prefer REST-style APIs, you might want to check out [Medusa](https://www.medusa-commerce.com/), which is similar to Vendure in many ways but uses a REST API.
